src/pages/Admin/Venue/Rooms/RoomsForm.tsx

```diff
@@ -319,30 +321,18 @@ const RoomInnerForm: React.FC<RoomInnerFormProps> = (props) => {
                     )}
                   </div>
                   <div className="toggle-room">
+                    {/* @debt pass the header into Toggler's 'label' prop instead of being external like this*/}
                     <div className="input-title">Enabled ?</div>
-                    <label className="switch">
-                      <input
-                        disabled={disable}
-                        type="checkbox"
-                        id="isEnabled"
-                        name={"isEnabled"}
-                        ref={register}
-                      />
-                      <span className="slider round"></span>
-                    </label>
+                    <Toggler
+                      name="isEnabled"
+                      forwardedRef={register}
+                      disabled={disable}
+                    />
                   </div>
                   <div className="toggle-room">
+                    {/* @debt pass the header into Toggler's 'label' prop instead of being external like this*/}
                     <div className="input-title">Is label hidden?</div>
-                    <label className="switch">
-                      <input
-                        disabled={disable}
-                        type="checkbox"
-                        id="isLabeled"
-                        name={"isLabeled"}
-                        ref={register}
-                      />
-                      <span className="slider round"></span>
-                    </label>
+                    <Toggler forwardedRef={register} disabled={disable} />
```

Comment:
This needs a `name`:
```suggestion
                    <Toggler name="isLabeled" forwardedRef={register} disabled={disable} />
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1775/commits/ccb242250c427352b560cd90eae05939aebeac49